apps/condo/domains/property/schema/Property.test.js

```diff
@@ -127,7 +120,5 @@ describe('Property', () => {
 
 })
 
-
-module.exports = {
-    makeClientWithProperty,
-}
\ No newline at end of file
+// To avoid circular dependencies
+module.exports.makeClientWithProperty = makeClientWithProperty
```

Comment:
It looks hacky

Reply:
I have removed it from Property.test.js to utils